tensorflow/contrib/tensorrt/convert/convert_graph.h

```diff
@@ -64,18 +64,6 @@ tensorflow::Status ConvertCalibGraphToInferGraph(
     const tensorflow::GraphDef& graph_def, tensorflow::GraphDef* new_graph_def,
     bool is_dyn_op);
 
-// - max_batch_size: maximum batch size which can be used for inference for
-//   optimization targets inference run with max batch size.
-// - max_workspace_size_bytes: The upper bound of memory allowance for engine
-//   building.
-tensorflow::Status ConvertGraphDefToTensorRT(
```

Comment:
Once this is removed, how a pure c/c++ application ask for a conversion without constructing whole grappler infrastructure, or at least a grappler cluster?

Reply:
If we don't need the graph_def, the simplest way would be to enable the trt optimizer and sess.run() will do the work. Otherwise we'll need to [create a cluster](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/grappler/cluster.i) and run [TF_OptimizeGraph](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/grappler/tf_optimizer.i#L95). I think if we need the graph, a more convenient way would be to run the tool to get the graph and then import it using c/c++.

Reply:
There are use cases of pure c++ applications using tftrt. Can you provide an example of doing conversion from a c++ API? Old API was straightforward to use.

Reply:
@aaroey, could you please provide a c++ only conversion example?

Reply:
Sorry for the delay, I'll rewrite ConvertGraphDefToTensorRT() using grappler later.